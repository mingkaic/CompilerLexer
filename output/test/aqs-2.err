output prefix: llvm/test/aqs-2
llvmas: /Users/cmk/llvm/bin/llvm-as
llc: /Users/cmk/llvm/bin/llc
cc: gcc
codegen: /Users/cmk/Developer/cworkspace/compiler/hw4/Lexer/answer/decafcomp
stdlib: /Users/cmk/Developer/cworkspace/compiler/hw4/Lexer/answer/decaf-stdlib.c
generating llvm code:/Users/cmk/Developer/cworkspace/compiler/hw4/Lexer/answer/decafcomp ... ok
; ModuleID = 'QuickSort'

@list = global [100 x i32] zeroinitializer

define void @main() {
entry:
  ret void
}
assembling to bitcode:/Users/cmk/llvm/bin/llvm-as "llvm/test/aqs-2.llvm" -o "llvm/test/aqs-2.llvm.bc" ... ok
converting to native code:/Users/cmk/llvm/bin/llc "llvm/test/aqs-2.llvm.bc" -o "llvm/test/aqs-2.llvm.s" ... ok
linking:gcc -o "llvm/test/aqs-2.llvm.exec" "llvm/test/aqs-2.llvm.s" "/Users/cmk/Developer/cworkspace/compiler/hw4/Lexer/answer/decaf-stdlib.c" ... ok
running:llvm/test/aqs-2.llvm.exec ... failed (144)
