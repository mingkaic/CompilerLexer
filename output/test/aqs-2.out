Program(None,Package(QuickSort,FieldDeclAST(list,IntType,Array(100)),Method(main,VoidType,None,MethodBlock(None,None))))
